cmd/tether/tether_linux.go

```diff
@@ -117,7 +117,7 @@ func (t *osopsLinux) setup() error {
 		log.Error(detail)
 		return errors.New(detail)
 	}
-	log.SetOutput(io.MultiWriter(out, os.Stdout))
+	log.SetOutput(io.MultiWriter(os.Stdout, out))
```

Comment:
curious why this was necessary - it will change the order of writing between VM console and debug log - was that somehow significant?